robot.py

```diff
@@ -1,15 +1,60 @@
+import ctre
 import wpilib
+from wpilib.interfaces import GenericHID
 
+from subsystems.drivetrain import Drivetrain
+from subsystems.elevator import Elevator
+from subsystems.grabber import Grabber
+from subsystems.wings import Wings
+
+LEFT_STICK = GenericHID.Hand.kLeft
+RIGHT_STICK = GenericHID.Hand.kRight
+
+
+# @TODO: Actually have motor IDs for these
+ELEVATOR_ID = 5
+LEFT_GRABBER_ID = 6
+RIGHT_GRABBER_ID = 7
+
+# !! These are the motor IDs for the Purple robot!
+# They may not be the correct ones for this year's competition robot
+LEFT1_ID = 3
+LEFT2_ID = 4
+RIGHT1_ID = 1
+RIGHT2_ID = 2
 
 class Robot(wpilib.IterativeRobot):
     def robotInit(self):
-        pass
+        left1 = ctre.WPI_TalonSRX(LEFT1_ID)
+        left2 = ctre.WPI_TalonSRX(LEFT2_ID)
+        left = wpilib.SpeedControllerGroup(left1, left2)
```

Comment:
It might be cleaner to just directly create all of these groups like?
```python
left = wpilib.SpeedControllerGroup(
    ctre.WPI_TalonSRX(LEFT1_ID),
    ctre.WPI_TalonSRX(LEFT2_ID),
)
```